index.html

```diff
@@ -30,6 +33,175 @@ <h2>I’am a Software Developer</h2>
             </div>
         </div>
     </section>
+    <!-- the hero section ends here -->
+
+    <!-- the works section -->
+    <section>
+        <div class="works-section">
+            <div class="recentwork">
+                <p>My Recent Works</p>
+            </div>
```

Comment:
- [ ] Dear @chibunduonyeje1, Please use a `hx (h2-h6)` tag for your `My Recent Works` instead of a `p (paragraph)` tag. This will provide a semantic wise advantage.